profile/deploy.yml

```diff
@@ -350,10 +349,11 @@ services:
       # Configure web server
       KC_HOSTNAME: ${OR_HOSTNAME:-localhost}
       KC_HOSTNAME_PORT: ${OR_SSL_PORT:--1}
-      KC_HOSTNAME_PATH: ${KC_HOSTNAME_PATH:-auth}
-      KC_HOSTNAME_ADMIN: ${KC_HOSTNAME_ADMIN:-}
-      KC_PROXY: ${KC_PROXY:-edge}
-
+      #KC_HOSTNAME_PATH: ${KC_HOSTNAME_PATH:-auth}
+      #KC_HOSTNAME_ADMIN: ${KC_HOSTNAME_ADMIN:-}
+      #KC_HTTP_ENABLED: ${KC_HTTP_ENABLED:-true}
+      #KC_PROXY_HEADERS: ${KC_PROXY_HEADERS:-xforwarded}
+      #KC_PROXY: ${KC_PROXY:-edge}
```

Comment:
The idea is to show common options that may need to be changed but a nicer way to handle this would be good as don't really want to replicate keycloak docker options unless necessary.